src/convert.js

```diff
@@ -15,7 +15,7 @@ module.exports = function (input, cb) {
     if (result.error) {
       return cb(null, {
         result: false,
-        reason: result.error.message
+        reason: result.error
```

Comment:
Will this break any existing contract? AFAIK we expect reason to be a `string` always from consumer side.

Reply:
We can add another field called `error` which can have the actual `error` and let `reason` be as it is. I checked NPM and there are no dependents on this library, so the only place we use it is the Postman App, where this is handled.

But yes, better to be backward compatible with interfaces.

Reply:
@dhwaneetbhatt Yes, we can have a separate error field rather than changing the reason.